src/app/(auth)/subscriber/create/page.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+
+import SubscriberForm from '@/components/Subscriber/SubscriberForm';
+import { UserGroupIcon } from '@heroicons/react/24/outline';
+import { createSubscriber } from '@/apis/subscriber';
+import { useState } from 'react';
+import { toast } from 'react-toastify';
+import { useRouter } from 'next/navigation';
+import { extractValuesFromOptions } from '@/utils/common';
+
+const Page = () => {
+  const router = useRouter();
+  const [selectedOptions, setSelectedOptions] = useState([]);
+
+  const onSubmit = async (values: any) => {
+    values.tags = extractValuesFromOptions(selectedOptions);
```

Comment:
instead of getting the values from the reactive property that's maintained inside this component, can't you get them directly from the `values`?
```js
values.tags = extractValuesFromOptions(values.tags)
```

Reply:
Yes, we can get the directly fromt he `values`. Made some changes could you please check.